ethportal-peertest/src/scenarios/offer_accept.rs

```diff
@@ -68,3 +72,55 @@ pub fn test_offer_accept(peertest_config: PeertestConfig, peertest: &Peertest) {
         "The received content {received_content_str}, must match the expected {HISTORY_CONTENT_VALUE}",
     );
 }
+
+pub async fn test_populate_offer(peertest_config: PeertestConfig, peertest: &Peertest) {
+    info!("Testing Populate Offer/ACCEPT flow");
+
+    let ipc_client = reth_ipc::client::IpcClientBuilder::default()
```

Comment:
This new way of testing jsonrpc methods over ipc differs from that in the previous test (`test_offer_accept`). I'd update the other test to match this new method, for congruency &  so it's easy to tell from reading the file that the same endpoint is being tested.